packages/sample/src/main.ts

```diff
@@ -715,7 +720,7 @@ program
         await runServo(hub, channelValue, pulseWidthValue, frameWidthValue);
     });
 
-program.parse(process.argv);
+await program.parseAsync(process.argv);
```

Comment:
I'm unclear on why this is here. Is parse not blocking?

Reply:
I actually didn't notice I was committing that, I'll get rid of it. This whole PR originated from my WebStorm Shelf.